po/pt-BR.po

```diff
@@ -6974,39 +7008,30 @@ msgid "The `_` pattern is a wildcard pattern which matches any value."
 msgstr "O padrão `_` é um padrão curinga que corresponde a qualquer valor."
 
 #: src/pattern-matching.md:26
-#, fuzzy
 msgid ""
 "* You might point out how some specific characters are being used when in a pattern\n"
 "  * `|` as an `or`\n"
 "  * `..` can expand as much as it needs to be\n"
 "  * `1..=5` represents an inclusive range\n"
 "  * `_` is a wild card\n"
-"* It can be useful to show how binding works, by for instance replacing a wildcard character with "
-"a variable, or removing the quotes around `q`.\n"
+"* It can be useful to show how binding works, by for instance replacing a wildcard character with a variable, or removing the quotes around `q`.\n"
 "* You can demonstrate matching on a reference.\n"
-"* This might be a good time to bring up the concept of irrefutable patterns, as the term can show "
-"up in error messages.\n"
+"* This might be a good time to bring up the concept of irrefutable patterns, as the term can show up in error messages.\n"
 "   "
 msgstr ""
-"<details>\n"
-"    \n"
-"Pontos chave:\n"
 "* Você pode apontar como alguns caracteres específicos podem ser usados em um padrão\n"
 "  * `|` como um `ou`\n"
 "  * `..` pode expandir o quanto for necessário\n"
 "  * `1..=5` representa um intervalo inclusivo\n"
 "  * `_` é um curinga\n"
-"* Pode ser útil mostrar como funciona a vinculação, por exemplo, substituindo um caractere curinga "
-"por uma variável ou removendo as aspas ao redor de `q`.\n"
+"* Pode ser útil mostrar como funciona a vinculação, por exemplo, substituindo um caractere curinga por uma variável ou removendo as aspas ao redor de `q`.\n"
 "* Você pode demonstrar correspondência em uma referência.\n"
-"* Este pode ser um bom momento para trazer à tona o conceito de padrões irrefutáveis, já que o "
-"termo pode aparecer em mensagens de erro.\n"
-"   \n"
-"</details>"
+"* Este pode ser um bom momento para trazer à tona o conceito de padrões irrefutáveis, já que o termo pode aparecer em mensagens de erro.\n"
+"   "
 
 #: src/pattern-matching/destructuring-enums.md:1
 msgid "# Destructuring Enums"
-msgstr "# Desestruturando enunerações (Enums)"
+msgstr "# Desestruturando Enumerações"
```

Comment:
Perhaps you could keep the term _Enums_ as it's used like this in the pt-BR Rust Book: https://rust-br.github.io/rust-book-pt-br/ch08-01-vectors.html?highlight=enum#usando-um-enum-para-armazenar-v%C3%A1rios-tipos